QUEST_JOBSTEP.tsv

```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
 QUEST_JOBSTEP_20150317_000078	Don't ask why. Work is work, and it's a fact that we don't want to see each other. 
 QUEST_JOBSTEP_20150317_000079	This pouch...Did you come from the Hunter Master? {nl}Here. Give her this book. 
 QUEST_JOBSTEP_20150317_000080	I don't want to act mean when she's acting humble. 
 QUEST_JOBSTEP_20150317_000081	This leather will be enough. {nl}Go to the Quarrel Shooter Master before someone else takes it. 
 QUEST_JOBSTEP_20150317_000082	This rock will be useful in repairing the wall. {nl}Bring it to the Archer Master quickly. 
-QUEST_JOBSTEP_20150317_000083	Are you the one who wants to learn my skills? {nl}Great, I was just looking for someone to help me find Lydia Schaffen's book. {nl}You're going to help right? 
-QUEST_JOBSTEP_20150317_000084	There is one in the Miner's Village but it was lost when the Vubbes raided. {nl} I'm sure it's in the Vubbe Outpost.
+QUEST_JOBSTEP_20150317_000083	$Are you the one who wants to learn my skills? {nl}Great, I was just looking for someone to help me find Lydia Schaffen's book. {nl}You're going to help, right? 
+QUEST_JOBSTEP_20150317_000084	$There is one in the Miners' Village, but it was lost when the Vubbes raided. {nl} I'm sure it's in the Vubbe Outpost.
 QUEST_JOBSTEP_20150317_000085	Oh yes, this is it. {nl}This will be helpful for people like you, who dream of becoming a Quarrel Shooter. {nl}I will give you a copy if you collect all the books. 
-QUEST_JOBSTEP_20150317_000086	Three more left. {nl}I think you can get one of them by giving some money to the Hunter Master..
+QUEST_JOBSTEP_20150317_000086	$Three more left. {nl}I think you can get one of them by giving some money to the Hunter Master.
 QUEST_JOBSTEP_20150317_000087	The problem is, I'm broke. 
 QUEST_JOBSTEP_20150317_000088	You mean Lydia Schaffen's book that I have? {nl}I'll hand it over for 5000 silver.
 QUEST_JOBSTEP_20150317_000089	Well then, thanks for your help. {nl}Around 5000 silver will do. {nl}
 QUEST_JOBSTEP_20150317_000090	5000 silver...Yup, that's right. {nl}I'll give you the book as promised. 
-QUEST_JOBSTEP_20150317_000091	Well...the Archer Master must have the next book. {nl}He's still helping repair the wall so wouldn't he appreciate it if you got him some stones? {nl}I could take it to him myself but...owwww...my back.
-QUEST_JOBSTEP_20150317_000092	The Archer Master has the next book. {nl}He's not a mean person so he will give you the book if you ask nicely. {nl}
-QUEST_JOBSTEP_20150317_000093	I heard he is helping repair the damaged city wall. {nl}There are rocks delivered to the Miner's Village so he'll be moved if you bring him that rock. 
+QUEST_JOBSTEP_20150317_000091	$Well...the Archer Master must have the next book. {nl}He's still helping repair the wall, so wouldn't he appreciate it if you got him some rocks? {nl}I could take them to him myself, but...owwww...my back.
```

Comment:
Having a "to" there feels more complete, maybe it's not grammatically incorrect to leave it out, but IMHO, it flows better with the "to" present.